src/components/SectionWizardView/types.ts

```diff
@@ -0,0 +1,25 @@
+import { ChipInterface } from "../Chip/chipInterface"
+import { ROFInterface } from "../ReadonlyField/rofInterface"
+
+interface IFields {
+    options: ROFInterface
```

Comment:
What is ROF, it should be meaning full name

Reply:
I have used existing types from Read only field component. Someone has already written it.